src/oaklib/implementations/simpleobo/simple_obo_implementation.py

```diff
@@ -809,11 +809,16 @@ def apply_patch(
             t = self._stanza(patch.about_node, strict=True)
             t.add_quoted_tag_value(TAG_DEFINITION, patch.new_value.strip("'"), xrefs=[])
             modified_entities.append(patch.about_node)
+        elif isinstance(patch, kgcl.RemoveTextDefinition):
+            t = self._stanza(patch.about_node, strict=True)
+            for tv in t.tag_values:
+                if tv.tag == TAG_DEFINITION:
+                    # This is a remove_definition request
+                    tv.value = ""
```

Comment:
the tag-val pair should be properly removed

Reply:
Done